.github/workflows/release.yml

```diff
@@ -23,10 +23,11 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install wasm-pack
         run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
-      - uses: Borales/actions-yarn@v2.3.0
+      - uses: actions/setup-node@v2
         with:
-          cmd: install
-      - name: Cache yarn build
+          node-version: "14"
```

Comment:
I don't know what you mean. Didn't we use yarn for all things related to Node until this PR?

Reply:
I remembered using Node 16 somewhere, and it turns out it's in our GitHub action: https://github.com/boa-dev/criterion-compare-action/blob/866adec9ed60aa872940c00b5adddff913fbc02e/.github/workflows/pr-main.yml

Reply:
Yeah we don’t want 14. We should be using 16 where we can, was there any particular reason to roll back?

Reply:
I see. As I've said in my previous comment, I opted to use v14 in order to facilitate contributions to the playground, since its package.json is more compatible with older versions of Node and we don't really need the newer features of v16.

Reply:
I can rollback to v16 if necessary, just note that Node v14 cannot build the playground with a `package-lock.json` generated with Node v16 (as the first few runs of the CI have showed)

Reply:
https://github.com/boa-dev/boa/runs/5588347786?check_suite_focus=true#step:7:15

Reply:
Hmm I think that should be ok, if someone got node onto their system they should be able to update it. V16 has been out for quite sometime now (v18 will be out in April)